aliases/available/general.aliases.bash

```diff
@@ -92,5 +59,6 @@ function catt() {
 # aliases and enable just the ones for Bash-it explicitly:
 # bash-it disable alias general
 # bash-it enable alias bash-it
-# shellcheck source-path=SCRIPTDIR
 source "$BASH_IT/aliases/available/bash-it.aliases.bash"
+source "$BASH_IT/aliases/available/directory.aliases.bash"
```

Comment:
loading it here wont make it load it twice in case its enabled?
we can check if the alias is already enabled, and `source` only if its not

Reply:
This one actually confused the heck out of me. If the user disabled it, we should not load it. If the user disabled it, we don't need to load it. How can we tell? No idea.

Reply:
hmmm, I think that we should always check here if enabled, enable and source if not. If users dont want a certain alias, they should disable the general file. We can add a log here about it. I do not want to break backwards compat with this change

Reply:
Sounds right to me too.